.gitignore

```diff
@@ -64,3 +64,4 @@ conf/
 *unit_test_base_tmp/
 
 */.gitignore
+local/conf/
```

Comment:
I tested locally and found that this directory can be ignored by the configuration on line 61, so there is no need to add additional information to the `.gitignore`.

```
➜  arctic git:(master) git check-ignore -v local/conf
.gitignore:61:conf/     local/conf
```

which means that the directory named `conf` in the root directory and all its subdirectories will be ignored.